site/partials/examples.html

```diff
@@ -30,6 +30,7 @@
 <li><code>domain</code>: x axis possible values (see examples below)</li>
 <li><code>yAxis</code>: same options as <code>xAxis</code></li>
 <li><code>disableZoom</code>: true to disable translation/scaling on the graph</li>
+<li><code>centerAxes</code>: true to center the axes in the middle of the graph</li>
```

Comment:
This file is autogenerated, first please address the API design comment and we can come back to this file later.